supertokens_python/recipe/dashboard/recipe.py

```diff
@@ -171,6 +177,8 @@ async def handle_api_request(
             api_function = handle_user_password_put
         elif request_id == USER_EMAIL_VERIFY_TOKEN_API:
             api_function = handle_email_verify_token_post
+        elif request_id == EMAIL_PASSSWORD_SIGNOUT:
+            api_function = handle_signout
```

Comment:
Maybe call it `handle_emailpassword_signout_api` (Keeping it consistent with signin)